quiz.js

```diff
@@ -31,12 +33,22 @@ var dingHandle = setTimeout (writeDing(), 3000);
 // Remember that if x is a variable containing a string, you can find its
 // length with the expression x.length.
 
+
 var words = ['short', 'medium', 'long', 'interminable', 'lengthy'];
+words.sort(function)
 
 var byLength = function (A, B) {
-  // your code here
+  if ((AB === true) && (BA === false) {
```

Comment:
you had the right idea here, but you need to compare the length of A and B instead of multiplying them together